sections/semantics-embedded-content.include

```diff
@@ -3623,12 +3623,63 @@
   For images that have captions, markup generators should use the
   <{figure}> and <{figcaption}> elements to provide the image's caption.
 
-  As a last resort, implementors should either set the <code>alt</code> attribute to the empty string, under
-  the assumption that the image is a purely decorative image that
-  doesn't add any information but is still specific to the surrounding
-  content, or omit the <code>alt</code> attribute
-  altogether, under the assumption that the image is a key part of the
-  content.
+  As a last resort, implementors should set the <code>alt</code> attribute to the empty
+  string, under the assumption that the image is a purely decorative image that doesn't add any
+  information but is still specific to the surrounding content, or omit the <code>alt</code>
+  attribute altogether, under the assumption that the image is a key part of the content.
+
+  <div class="note">
+    Setting an <{image}> element's <code>alt</code> attribute to the empty string will convey
+    to assistive technologies, such as screen readers, that the image in question has no relative
+    or informative value to the section of the document in which it is found. Being that it serves
+    no informative purpose, a screen reader may choose to completely ignore the presence of the
```

Comment:
Suggest changing this to "a screen reader will..." instead of "a screen reader may...", since all screen readers will completely ignore an image with an empty alt attribute.